Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3375,10 +3379,12 @@ else if (thisReq_ instanceof ServerRequestLogout) {
                                     thisReq_.onRequestSucceeded(serverResponse, branchReferral_);
                                 }
                             }
+                            BroadcastSessionState(initState_);
```

Comment:
Wondering if this should perhaps be moved up into the `if (thisReq_ instanceof ServerRequestInitSession)` block - since that's the only time initState is updated to initialized? just a thought, would want to prevent excess broadcasts